README.md

```diff
@@ -1,43 +1,50 @@
-# <a href='https://www.apollographql.com/'><img src='https://user-images.githubusercontent.com/841294/53402609-b97a2180-39ba-11e9-8100-812bab86357c.png' height='100' alt='Apollo Feature Requests'></a>
+<a href='https://www.apollographql.com/'><img src='https://user-images.githubusercontent.com/841294/53402609-b97a2180-39ba-11e9-8100-812bab86357c.png' height='100' alt='Apollo Feature Requests'></a>
 
-## Apollo Feature Requests
+# 👋 Welcome
 
-This repository is used to track [Apollo Client](https://github.com/apollographql/apollo-client) and [React Apollo](https://github.com/apollographql/react-apollo) feature requests and discussions. Click [here](https://github.com/apollographql/apollo-feature-requests/issues/new) to open a new feature request.
+Many of the features in the Apollo Platform came from suggestions by you, the community! We welcome any ideas about how to make the Apollo Core Platform better for your use cases. Unless there is overwhelming demand for a feature, it might not get implemented immediately, but please include as much information as possible that will help people have a discussion about your proposal.
```

Comment:
```suggestion
Many of the features in the core Apollo projects came from suggestions by you, the community! While each project has it's own repo for tracking issues, we welcome any ideas about how to make the end-to-end supergraph experience across the core projects work better for your use cases. Unless there is overwhelming demand for a feature, it might not get implemented immediately, but please include as much information as possible that will help people have a discussion about your proposal.
```